doc/manuals/contribution_guidelines.md

```diff
@@ -405,6 +405,17 @@ directly or indirectly are forced to have that construct as well.
 
 *How to check*: using an intelligent ```grep```.
 
+#### M12bis (`using namespace` in source code files)
+
+*Rule 1*: the construct ```using namespace XXX``` must not be used in source code files.
+*Rule 2*: the construct ```using XXX::YYY" **may** be used in source code files, but even better is to **not** use `using` but to write out the namespace (call it prefix if you want) wherever needed.
```

Comment:
> but even better is to **not** use `using` but to write out the namespace (call it prefix if you want) wherever needed

That's subjetive... I guess @kzangeli think this is important (as his code uses that), @fgalan doesn't think is so important (as his code uses that). 

Let's remove that part and make that a free choice for the programmer. I mean, the program chose either `ns::myFunct()`` or `using ns::myFunc` + `myFunc()`.